src/Compilers/Core/Portable/Emit/EditAndContinue/EncVariableSlotAllocator.cs

```diff
@@ -265,7 +265,7 @@ private bool TryGetPreviousLambdaSyntaxOffset(SyntaxNode lambdaOrLambdaBodySynta
             SyntaxNode previousSyntax;
             if (isLambdaBody)
             {
-                previousSyntax = previousLambdaSyntax.TryGetCorrespondingLambdaBody(lambdaOrLambdaBodySyntax);
+                previousSyntax = TryGetCorrespondingLambdaBody(previousLambdaSyntax, lambdaOrLambdaBodySyntax);
```

Comment:
@tmat This was the only call to .TryGetCorrespondingLambdaBody.  So it seems like TryGetCorrespondingLambdaBody was very tied to the EncVariableSlotAllocator